autoload/ale/handlers/languagetool.vim

```diff
@@ -0,0 +1,62 @@
+" Author: Vincent (wahrwolf [ät] wolfpit.net)
+" Description: languagetool for markdown files
+
+function! ale#handlers#languagetool#HandleOutput(buffer, lines) abort
+    " Match lines like:
+    " 1.) Line 5, column 1, Rule ID:
+    let l:head_pattern = '^\v.+.\) Line (\d+), column (\d+), Rule ID. (.+)$'
+    let l:head_matches = ale#util#GetMatches(a:lines, l:head_pattern)
+
+    " Match lines like:
+    " Message: Did you forget a comma after a conjunctive/linking adverb?
+    let l:message_pattern = '^\vMessage. (.+)$'
+    let l:message_matches = ale#util#GetMatches(a:lines, l:message_pattern)
+
+    " Match lines like:
+    "   ^^^^^ "
+    let l:markers_pattern = '^\v *(\^+) *$'
+    let l:markers_matches = ale#util#GetMatches(a:lines, l:markers_pattern)
+
+    let l:output = []
+
+
+    " Okay tbh I was to lazy to figure out a smarter solution here
+    " We just check that the arrays are same sized and merge everything
+    " together
+    let l:i = 0
+
+    while l:i < len(l:head_matches) &&
+    \   (
+    \       (len(l:head_matches) == len(l:markers_matches)) &&
```

Comment:
Move the binary operators to the start of the following line. The linting tools will complain about this now.

Reply:
Done!